settings/security/third-party/saml.rst

```diff
@@ -19,81 +22,78 @@ Configure SAML in Zammad
-
-``urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress``
-
-
-Configured example:
-
-.. image:: /images/settings/security/third-party/saml/zammad_connect_saml_thirdparty.png
-   :alt: Zammad SAML Example
-
+   ``urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress``
 
 Configure the SAML IdP
 ----------------------
 
 If your IdP supports xml import or auto xml metadata retrieval, use this URL:
-``https://zammadFQDN/auth/saml/metadata``
+``https://{Zammad-FQDN}/auth/saml/metadata``
 
 For Keycloak IdP, save the contents of that metadata file as "metadata.xml" and then log into Keyclaok and import it as a new Client.
 
 Keycloak sends the "Username" in whatever format your users are stored in. The tested SAML instance used email address for the "username", but Zammad also needs to see an "email" attribute to correctly automap to existing users.
 In Keycloak, we can create a "Mapper" for the newly imported Zammad client.
 
-Goto -> Keycloak Admin -> Clients -> ``https://zammadFQDN/auth/saml/metadata`` -> Mappers -> Create
+Goto -> Keycloak Admin -> Clients -> ``https://{Zammad-FQDN}/auth/saml/metadata`` -> Mappers -> Create
 
 .. code-block:: html
 
    Name: ZammadEmail
-   Mapper Type: User Attribute
+   Mapper Type: User Property
    User Attribute: emailAddress
    SAML Attribute Name: email
    SAML Attribute NameFormat: Basic
 
 Keycloak needs to know the redirect location in advance for security:
-Goto -> Keycloak Admin -> Clients -> ``https://zammadFQDN/auth/saml/metadata``
-
-**Valid Redirect URIs:**
+Goto -> Keycloak Admin -> Clients -> ``https://{Zammad-FQDN}/auth/saml/metadata``
 
-``https://zammadFQDN/auth/saml/callback``
+Valid Redirect URIs:
+   ``https://{Zammad-FQDN}/auth/saml/callback``
 
 For other IdP systems, you can configure things manually.
 Zammad is using POST Bindings for the Assertion Consumer Service (ACS)
 
+ASC POST Binding URL:
+   ``https://{Zammad-FQDN}/auth/saml/callback``
 
-**ASC POST Binding URL:**
-
-``https://zammadFQDN/auth/saml/callback``
+Zammad requests these attributes from the SAML IdP, these are not configurable. Each attribute should map to the correct 
+SAML attribute via "Mappers" as needed, or equivalent mapping paradigm in other IdP systems.
+The specific attributes to map vary widely between SAML IdP systems, this requested attribute list can help align the SAML 
+provided attributes with what Zammad expects to see.
 
-Zammad requests these attributes from the SAML IdP, these are not configurable. Each attribute should map to the correct SAML attribute via "Mappers" as needed, or equivalent mapping paradigm in other IdP systems.
-The specific attributes to map vary widely between SAML IdP systems, this requested attribute list can help align the SAML provided attributes with what Zammad expects to see.
+   .. warning:: Please note that attribute may mean property in some cases! Mappings with user properties should be 
+      what you're aiming for.
 
 .. code-block:: html
 
```

Comment:
I've rephrased this entire section thusly:

![Screenshot_20200728_232715](https://user-images.githubusercontent.com/12194123/88686880-1e92f300-d12a-11ea-9364-9d5ed32879c8.png)

Is that a reasonable/accurate rewriting of this section?

Reply:
Looks good to me, I'd prefer to keep
```
   .. warning:: Please note that attribute may mean property in some cases! Mappings with user properties should be what you're aiming for.
```

Because this was added during a support case I believe.

Reply:
This phrasing okay?

> Zammad requests the following user attributes (or “properties”) from the IdP...